Modules/Utils/Sources/PocketCastsUtils/Extensions/UI/UITextView+scrollToRange.swift

```diff
@@ -0,0 +1,35 @@
+#if !os(watchOS)
+import UIKit
+
+extension UITextView {
+    public func scrollToRange(_ range: NSRange) {
+        // Ensure layout is up-to-date
+        textStorage.addLayoutManager(layoutManager)
+        layoutManager.ensureLayout(for: textContainer)
+
+        let glyphRange = layoutManager.glyphRange(forCharacterRange: range, actualCharacterRange: nil)
+
+        layoutManager.enumerateEnclosingRects(forGlyphRange: glyphRange, withinSelectedGlyphRange: NSRange(location: NSNotFound, length: 0), in: textContainer) { rect, _ in
```

Comment:
Just double check if a retain cycle happens here as the `enumerateEnclosingRects` closure is `escaping`

Reply:
Good catch! Fixed in 1999d3f868649eff31832c3dbf2805ef918a799a